Schema.js

```diff
@@ -75,6 +78,7 @@ function classNameIsValid(className) {
     className === '_Session' ||
     className === '_SCHEMA' || //TODO: remove this, as _SCHEMA is not a valid class name for storing Parse Objects.
     className === '_Role' ||
+    className === '_GlobalConfig' ||
```

Comment:
Or I can just do `rawCollection().findAndModify()` on it if that's ok. I'll update my PR with that and wait for your comments.